sdk/python/feast/field.py

```diff
@@ -65,6 +71,13 @@ def dtype(self) -> ValueType:
         """
         return self._dtype
 
+    @property
+    def labels(self) -> MutableMapping[str, str]:
+        """
+        Getter for labels of this field
+        """
+        return self._presence
```

Comment:
Copypasta mistake, should be `_labels` 😇 

Calls to attention that we're missing label tests for fields, although I wonder if we should just be removing `Field` from the Python SDK at this point.

Reply:
Good catch. Changed it and added some test cases.